lib/availability.ts

```diff
@@ -45,3 +52,39 @@ export function getAvailabilityFromSchedule(schedule: Schedule): Availability[]
     return availability;
   }, [] as Availability[]);
 }
+
+/**
+ * PostgreSQL stores the date relatively
+ */
+export function getWorkingHours(
+  timeZone: string,
+  availability: Pick<Availability, "days" | "startTime" | "endTime">[]
+) {
+  // clearly bail when availability is not set, set everything available.
+  if (!availability.length) {
+    return [
+      {
+        days: [0, 1, 2, 3, 4, 5, 6],
+        // shorthand for: dayjs().startOf("day").tz(timeZone).diff(dayjs.utc().startOf("day"), "minutes")
+        startTime: 0,
+        endTime: 1439,
+      },
+    ];
+  }
+
+  const workingHours = availability.map((schedule) => ({
+    days: schedule.days,
```

Comment:
This is where things go wrong, say you are in belgium and your time is 00:00:00 - 20:00:00 & you are only available on Monday. when this is cast to UTC what needs to happen is the creation of two workingHour objects. Belgium UTC+1

```
[ {
  days: [ 0 ],
  startTime: 1380 // 23:00:00
  endTime: 1439 // 23:59:59
}, {
  days: [ 1 ],
  startTime: 0, 
  endTime: 1139
} ]

// currently:
[ {
  days: [ 1 ],
  startTime: -60, 
  endTime: 1139
} ]
```